extensions/sql-database-projects/src/test/project.test.ts

```diff
@@ -947,6 +947,46 @@ describe('Project: Msbuild sdk style project content operations', function (): v
 		// <Build Remove="file1.sql" />
 		should(project.files.filter(f => f.relativePath === 'file1.sql').length).equal(0);
 	});
+
+	it('Should only add Build entries to sqlproj for files not included by project folder glob and external streaming jobs', async function (): Promise<void> {
+		projFilePath = await testUtils.createTestSqlProjFile(baselines.openNewStyleSqlProjectBaseline);
```

Comment:
Is there a more official name than "newStyleSqlProj" that we could use?  This might get confusing down the road.

Reply:
oh yeah that isn't very clear. I thought I had named these "newMsbuildStyle..." so I'll update them

Reply:
Updated the names in this PR: #17677. If there's a more official name instead of "msbuild sdk style project" I can update it to that.